.vitepress/constants/constants.js

```diff
@@ -1,7 +1,7 @@
 const constants = Object.freeze({
   golangNodeMainnet: "1.22.0",
-  golangNodeMocha: "1.22.0",
-  golangNodeArabica: "1.22.0",
+  golangNodeMocha: "1.23.0",
+  golangNodeArabica: "1.23.0",
   golangApp: "1.22.0",
   golangCore: "1.22.0",
```

Comment:
@jcstein these use Go 1.22.0 which work with celestia-app v2.1.2 b/c https://github.com/celestiaorg/celestia-app/blob/48173df3dc78f9348eedb3796f29ef9e9e5dc584/go.mod#L3

We could also do this

```suggestion
  golangApp: "1.22.6",
  golangCore: "1.22.6",
```

Reply:
Build failed with 1.22 on node v0.16.0. Tagging @walldiss for review from node team to recommend either 1.23 or 1.22.6

Reply:
Can these diverge? My preference based on go.mod would be:

```
  golangNodeMocha: "1.23.0",
  golangNodeArabica: "1.23.0",
  golangApp: "1.22.6",
  golangCore: "1.22.6",
```

- https://github.com/celestiaorg/celestia-node/blob/6744f648649ebb5fee1b27faf7aca96ecf4519b2/go.mod#L3
- https://github.com/celestiaorg/celestia-app/blob/48173df3dc78f9348eedb3796f29ef9e9e5dc584/go.mod#L3

Reply:
Yep, this should work! Good suggestion